sbi/utils/torchutils.py

```diff
@@ -207,12 +207,16 @@ def __init__(
         super().__init__(Uniform(low=low, high=high), reinterpreted_batch_ndims)
 
 
-# XXX does an in-place version (e.g. make_conform_) make sense?
+# XXX does an in-place version (e.g. ensure_parameter_batched_) make sense?
 def ensure_parameter_batched(parameter: torch.Tensor) -> torch.Tensor:
     """
-    Return tensors that both have the same tensor.ndim
-    Function also covers cases where parameters is ndim=1 and observation ndim=2
-    Also, we have specialized check for multi-dimensional data x, e.g. images.
+    Return parameter tensor that has a batch dimension, i.e. has shape
+    (1, dim_parameters)
+
+    Args:
+        parameter: A single parameter set, potentially without batch dimension
```

Comment:
If we are not going to relax parameter to be an `iterable` (much less a Python `Set`) I would directly mention here a Tensor, possibly along the following lines: a tensor of n real parameters of shape (n,) or (1,n).